Makefile

```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
 	@yarn run test-ci
 	@echo ""
 
-test-python:
-	@echo "--> Running Python tests"
-	# This gets called by getsentry
-	pytest tests/integration tests/sentry
-
 test-python-ci:
-	make build-platform-assets
 	@echo "--> Running CI Python tests"
 	pytest tests/integration tests/sentry \
 		--ignore tests/sentry/eventstream/kafka \
 		--ignore tests/sentry/snuba \
 		--ignore tests/sentry/search/events \
+		--ignore tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
```

Comment:
Commenting for helping my own understanding.

This test is skipped here because it's been xfailing.
> reason: test requires SENTRY_KAFKA_HOSTS environment variable which is not set